doc/dao/Index.md

```diff
@@ -0,0 +1,44 @@
+# Contributing as part of the DAO
+
+When contributing to Primero, either as part of the DAO, or as a traditional open-source contribution, it is necessary to follow a set of guidelines. This helps ensure that any new contributions meet existing code standards, and are as likely as possible to be merged into the upstream repository.
+
+In these instructions, we walk through the steps that are required to add a new feature to Primero.
+
+If you are contributing as part of the DAO, you first need to create a proposal, which will be voted upon by the DAO members for approval. 
+
+You will then work on your feature and submit a Pull Request to the Primero GitHub Repository.
+
+> [!WARNING]
+> Even if your proposal is approved by the DAO, it is still subject to code review by the Primero development team. This is a mandatory step, and you will not receive any payout if you do not pass code review. The Primero development team is not obligated to accept any proposed changes, even if the DAO has approved the proposal.
+
+The following flow chart illustrates the overall process for contributing via the DAO and receiving payment for the associated work.
+
+```mermaid
+flowchart TD
+
+Proposal(Create proposal)
+--> RefineProposal(Refine proposal)
+--> Vote1{{DAO votes on proposal}}
+-- Approved --> Begin(Work begins on proposed feature)
+Vote1 -- Rejected --> ProposalRejected([Proposal rejected])
+Begin --> PR(Pull Request opened)
+--> DevTeamReviews{{Primero Development Team reviews code}}
+DevTeamReviews -- Dev Team Proposes Changes --> ImplementChanges(Proposed changes are implemented) --> UpdatePR(Pull Request updated with new changes) --> DevTeamReviews
+DevTeamReviews -- Dev Team Approves PR --> Report(Report completion of proposal to DAO)
+DevTeamReviews -- Dev Team Rejects PR --> CodeRejected([Code rejected])
+
+Report --> Verify(DAO verifies that the proposal has been satisfied by the PR)
+--> Vote2{{DAO votes on completed proposal}}
+-- DAO Satisfied --> Paid([Funds Paid])
```

Comment:
This is good from a dev process but we are missing some things here:
- Where/How/Who will verify that the proposal meets acceptance from the users (UAT).
- Similar questions for QA: Code reviews and unit tests are all well and good but how will we know that the change is fundamentally sound and  didn't introduce instability in the rest of the system? 
- Will there be a server? Do the developers need to stage their code somewhere?

Reply:
What I am worried about here is a merge into the develop branch that is unsatisfactory that will hold up the release of other perfectly good features

Reply:
These are great questions. I don't know how best to handle UAT or QA for any contributions coming through the DAO. I think that it will really depend on what kinds of changes people actually propose. 

> Will there be a server? Do the developers need to stage their code somewhere?
Should I say that a PR needs to have a link to a server running any new features as part of the checklist?

Reply:
> Should I say that a PR needs to have a link to a server running any new features as part of the checklist?

That's a hell of an ask for someone doing a low budget DAO feature IMO